### DISCUSSION TITLE
doubt with the take profit

### DISCUSSION THREAD

**Discussion by Brianramirez27** [kernc/backtesting.py#479] [General]
Hello, what happens is that when I establish the take porfi and verify the closing, it closes very late, the price reaches the point of the tp but it does not close when the price returns, why will it be? I attach an image to show my doubt
![pregunta101](https://user-images.githubusercontent.com/84251018/134524522-9a493787-44e0-4fbc-abb9-89ca94b89a5c.JPG)

**Comment by kernc** [comment]
Can you show a little bit of code? If that doesn't work, surely you can make a quick example using `backtesting.lib.GOOG` data?

**Reply by Brianramirez27** [reply]
![vbvbgfhf](https://user-images.githubusercontent.com/84251018/135117443-a6d8bede-a24d-49c4-89a3-f9e11f705aa3.JPG)

this is the code
